bindings/matrix-sdk-crypto-js/tests/device.test.js

```diff
@@ -538,9 +537,9 @@ describe("Key Verification", () => {
 
         test("can confirm back keys match (`m.key.verification.done`)", async () => {
             // `m2` confirms.
-            const [outgoingVerificationRequests, signatureUploadRequest] = await sas2.confirm();
+            const outgoingVerificationRequests = await sas2.confirm();
 
-            expect(signatureUploadRequest).toBeUndefined();
+            // there should be a m.key.verification.mac and a m.key.verification.done, but no SignatureUploadRequest
```

Comment:
Can you add a test that contains a signature upload request too, please?

Reply:
it wasn't easy, but yes!